dev/tests/functional/tests/MFTF/DevDocs/Test/DevDocsTest.xml

```diff
@@ -22,5 +22,20 @@
         <!--   Open MFTF DevDocs Page -->
         <amOnPage stepKey="openMFTFDevDocPage" url="{{MFTFDocPage.url}}" />
         <see stepKey="verifyPageIntroText" selector="{{contentSection.pageIntro}}" userInput="{{MessageData.message}}" />
+        <helper class="\MFTF\DevDocs\Helper\CustomHelper" method="goTo" stepKey="customHelper">
+            <argument name="test">{{contentSection.pageIntro}}</argument>
+            <argument name="module">['Test', 'Bla']</argument>
+            <argument name="url">{{MFTFDocPage.url}}</argument>
+        </helper>
+
+        <helper class="\MFTF\DevDocs\Helper\CustomHelper" method="goTo" stepKey="customHelperWithArrayParametrized">
+            <argument name="test">{{contentSection.pageIntro}}</argument>
+            <argument name="module">['{{DeprecatedContentSection.pageIntro}}', 'Bla']</argument>
+            <argument name="url">{{DeprecatedMFTFDocPage.url}}</argument>
+        </helper>
+
+        <actionGroup ref="HelperActionGroup" stepKey="actionGroupWithCustomHelper">
+            <argument name="test" value="{{HelperData.entityField}}" />
```

Comment:
HelperData is not found, needs to be checked in?

Reply:
Fixed

Reply:
Please pull latest change. I forgot to git add `HelperActionGroup.xml`